assets/js/app.js

```diff
@@ -0,0 +1,25 @@
+const hambger = document.querySelector('.hambuger');
+const menu = document.querySelector('.menu');
+const closebutton = document.querySelector('.close');
+const logo = document.querySelector('.logo');
+
+hambger.addEventListener('click', () => {
+  menu.style.display = 'block';
+  closebutton.style.display = 'block';
+  hambger.style.display = 'none';
+  logo.style.display = 'none';
+});
+
+closebutton.addEventListener('click', () => {
+  menu.style.display = 'none';
+  closebutton.style.display = 'none';
+  hambger.style.display = 'block';
+  logo.style.display = 'block';
+});
+
+menu.addEventListener('click', () => {
+  menu.style.display = 'none';
+  closebutton.style.display = 'none';
+  hambger.style.display = 'block';
+  logo.style.display = 'block';
+});
```

Comment:
- [OPTIONAL] Great job here 💯 But your code could be better, because I repeatedly noticed in line 14:17 that it's exactly the same from line 21:24, So to follow the best practice for [ JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) and Keep your [code clean](https://devinduct.com/blogpost/22/javascript-clean-code-best-practices) and DRY. I suggest making two functions for close and open menu.

> For example 💡
```
const openMenu = () => {
  menu.style.display = 'block';
  closebutton.style.display = 'block';
  hambger.style.display = 'none';
  logo.style.display = 'none';
};

const closeMenu = () => {
  menu.style.display = 'none';
  closebutton.style.display = 'none';
  hambger.style.display = 'block';
  logo.style.display = 'block';
};

hambger.addEventListener('click', openMenu);
closebutton.addEventListener('click', closeMenu);
menu.addEventListener('click', closeMenu);
```

Reply:
Wow 👍 
Thank you for this great piece of advice